const product=async(req,res)=>{
    try{

    }
    catch(error){
        res.status(400).send({message:error.messge})
    }

}